.github/ISSUE_TEMPLATE/1_bug_report.yml

```
@@ -5,11 +5,12 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thank you for taking the time to fill out this bug report. Please take the time to provide as much details as possible. Following our [bug report guidelines](https://devdocs.prestashop-project.org/8/contribute/contribute-reporting-issues/#best-practices-for-writing-an-issue) will help maintainers qualify your report faster.
-
-        Reporting bugs with "error 500" or "white page" may not help us reproduce them. This is a universal error that hides what is underneath. Instead, enable debug mode in the Performance section of PrestaShop and try again. You should get a proper error message.
-
-        🇬🇧 Use English, reports in all other languages will be closed.
+         #### Basic information ❗
+         - Provide as much DETAILS as possible - error logs, screenshots, your exact configuration. If no one can reproduce it, no one will fix it.
```

Pick one:
- As much detail
- As many details

```suggestion
         - **Provide as much detail as possible** - error logs, screenshots, your exact configuration. If the issue cannot be reproduced, it cannot be fixed.
```